rust_icu_ecma402/src/listformat.rs

```diff
@@ -0,0 +1,234 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Implements the traits found in [ecma402_traits::listformat].
+
+use ecma402_traits;
+use ecma402_traits::listformat;
+use rust_icu_common as common;
+use rust_icu_ulistformatter as ulfmt;
+use std::fmt;
+
+/// Implements [listformat::Format] using ICU as the underlying implementing library.
+#[derive(Debug)]
+pub struct Format {
```

Comment:
I would name it `ListFormat`

Reply:
Same as above, without an explanation as to why,  this is one personal preference vs another.  I'll resolve this conversation as I expect the rationale and decision will be the same as above.